homeassistant/components/homematic/__init__.py

```diff
@@ -573,6 +573,8 @@ def _create_ha_id(name, channel, param, count):
     if count > 1 and param is not None:
         return f"{name} {channel} {param}"
 
+    raise ValueError(f"Unable to create unique id for count:{count} and param:{param}")
```

Comment:
I'm unsure about this one.
Currently it returns `None`, but I wonder if the whole logic should be adjusted...
Can count ever be less than 1 ?

Reply:
Yeah, I would not change the logic like this. The integration have quite a bit work left (including a different client library). Let's not touch to much of that now